fixtures/js-stubs/authProviders.tsx

```diff
@@ -0,0 +1,17 @@
+import type {AuthProvider as AuthProviderType} from 'sentry/types';
```

Comment:
wouldn't it be better if we don't rename the import?

Reply:
Yes, but we cant because it matches the stub value, so we need to alias the import